tests/cephadm/test_cephadm_mgr_timeout.py

```diff
@@ -0,0 +1,76 @@
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError, UnexpectedStateError
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify cephadm command timeout
+    Args:
+        **kw: Key/value pairs of configuration information
+              to be used in the test.
+    """
+    admin = ceph_cluster.get_nodes(role="_admin")[0]
+
+    # Set default_cephadm_command_timeout to 120 seconds
+    CephAdm(admin).ceph.config.set(
+        key="mgr/cephadm/default_cephadm_command_timeout", value="120", daemon="mgr"
+    )
+    log.info("Default cephadm command timeout set to 120 seconds")
+
+    # Download the 'cephadm-hold-lock.py' script to the admin node
+    repo_url = "https://raw.githubusercontent.com/adk3798/testing_scripts/main/cephadm-hold-lock.py"
```

Comment:
Why this external utility is used ? Is it approved for use?

Reply:
This scipt might get deleted. It's not a good idea to depend on this utility.

Reply:
I've added the script to cli/utilities so that there is no dependency on external repos.